src/components/date-input/index.njk

```diff
@@ -133,6 +133,34 @@
       {
         text: 'Arguments for the fieldset component (e.g. legend). See fieldset component.'
       }
+    ],
+    [
```

Comment:
I think we might need `hint` object as well.

Reply:
added